src/components/RouteUpdate.tsx

```diff
@@ -0,0 +1,300 @@
+                    onClose={onCancel}
+                    autoFocus={true}
+                    footer={
+                        <Space style={{display: 'flex', justifyContent: 'end'}}>
+                            <Button onClick={onCancel} disabled={savedRoute.loading}>Cancel</Button>
+                            <Button type="primary" disabled={savedRoute.loading} onClick={handleFormSubmit}>{`${formRoute.id ? 'Save' : 'Create'}`}</Button>
+                        </Space>
+                    }
+                >
+                    <Form layout="vertical" hideRequiredMark form={form} onValuesChange={onChange}>
+                        <Row gutter={16}>
+                            <Col span={24}>
+                                <Header style={{margin: "-32px -24px 20px -24px", padding: "24px 24px 0 24px"}}>
+                                    <Row align="top">
+                                        <Col flex="none" style={{display: "flex"}}>
+                                            {!editName && !editDescription && formRoute.id  &&
+                                                <button type="button" aria-label="Close" className="ant-drawer-close"
+                                                        style={{paddingTop: 3}}
+                                                        onClick={onCancel}>
+                                                    <span role="img" aria-label="close" className="anticon anticon-close">
+                                                        <CloseOutlined size={16}/>
+                                                    </span>
+                                                </button>
+                                            }
+                                        </Col>
+                                        <Col flex="auto">
+                                            { !editName && formRoute.id ? (
+                                                <div className={"access-control input-text ant-drawer-title"} onClick={() => toggleEditName(true)}>{formRoute.id ? formRoute.network_id : 'New Route'} Network</div>
+                                            ) : (
+                                                <Form.Item
+                                                    name="network_id"
+                                                    label="Network Identifier"
+                                                    tooltip="You can combine an identifier with a Network CIDR to form a high availability route"
+                                                    rules={[{required: true, message: 'Please add an identifier for this access route', whitespace: true}]}
+                                                >
+                                                    <Input placeholder="Add network identifier..." ref={inputNameRef} onPressEnter={() => toggleEditName(false)} onBlur={() => toggleEditName(false)} autoComplete="off" maxLength={40}/>
```

Comment:
When the network identifier is set,  the UI shows this:
`my-identifier Network`

Should the "Network" word be there or maybe it should be lowercase?